src/gui/editors/TimeLineWidget.cpp

```diff
@@ -273,9 +273,11 @@ void TimeLineWidget::paintEvent( QPaintEvent * )
 	for( int i = 0; x + i * m_ppb < width(); ++i )
 	{
 		++barNumber;
+		// round m_ppb to the nearest power of two
+		float ppbP2 = pow(2, round(log2(m_ppb)));
 		if( ( barNumber - 1 ) %
 			qMax( 1, qRound( 1.0f / 3.0f *
-				TimePos::ticksPerBar() / m_ppb ) ) == 0 )
+				TimePos::ticksPerBar() / ppbP2 ) ) == 0 )
```

Comment:
There are some code style issues here which could be fixed

Reply:
I was only modifying the name of the variable and didn't change original code. Being that line of code part of a bigger instruction, should I modify the whole instruction (lines 278, 279 and 280) or only the one that is involved with the new code? I think that should be the right line of action.
As for the name of the new variable ppbP2 (stands for "pixels per bar at power of 2") I'm not sure what is the best coding convection here: pppP2 or pPBP2 (I just followed the existing variable name)

Reply:
I'd go for modifying the whole instruction and let the variable name as it is.